ui/lib/utils/reportData.ts

```diff
@@ -0,0 +1,17 @@
+import mixpanel from 'mixpanel-browser'
+import client from '@lib/client'
+
+export async function init() {
+  mixpanel.init('cb1135f29a413c653332990f07b3586a', {
+    opt_out_tracking_by_default: true,
+  })
+  // check option
+  const res = await client.getInstance().getInfo()
+  if (res?.data?.enable_report) {
+    mixpanel.opt_in_tracking()
+  }
+}
```

Comment:
**In next PR**, we need to further identify users by using https://developer.mixpanel.com/docs/javascript#managing-user-identity

Notice that the identity in our case should comes from unique cluster uuid + the hash of user id (since different clusters will have the same user, e.g. root, but they are different users)